main.py

```diff
@@ -226,6 +231,27 @@ def string2timestamp(s):
 def year2timestamp(y):
     return time.mktime(datetime.date(y, 1, 1).timetuple())
 
+@app.route('/newFeatures', methods=['POST'])
+def updatebyemail():
+    emailaddress = str(request.json['address'])
+    if len(emailaddress)>40:
+        return jsonify(respo='Email too long', emailaddress = emailaddress)
+    try:
+        con = sqlite3.connect('emailAddresses.db')
```

Comment:
Thanks. I think it's much simpler and there's no reason to use a separate
DB. I don't think it's so bad if we create a user for each subscriber;
later we can support more fields and let them fill them.
All you need, I think, is a Subscription model with a foreign key to the
User model, and the relevant fields.
I can help some more next week if necessary.
On May 7, 2015 11:22 PM, "aviklai" <notifications@github.com> wrote:

> In main.py
> <https://github.com/hasadna/anyway/pull/242#discussion_r29891962>:
>
> > @@ -226,6 +231,27 @@ def string2timestamp(s):
> >  def year2timestamp(y):
> >      return time.mktime(datetime.date(y, 1, 1).timetuple())
> >
> > +@app.route('/newFeatures', methods=['POST'])
> > +def updatebyemail():
> > +    emailaddress = str(request.json['address'])
> > +    if len(emailaddress)>40:
> > +        return jsonify(respo='Email too long', emailaddress = emailaddress)
> > +    try:
> > +        con = sqlite3.connect('emailAddresses.db')
>
> I need to save in the database only the email address and in the users
> database there are other details as well. About the FLASK ORM - the reason
> I used pure sqlite is that I am not familiar with SQLAlchemy. I will read
> about SQLAlchemy and try to implement it with it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hasadna/anyway/pull/242/files#r29891962>.
>